test/specs/behaviors/testHelper.tsx

```diff
@@ -96,4 +117,20 @@ export class TestHelper {
     }
     return stringToConvert
   }
+
+  private verifySpecificationTag(behaviorFileName: string) {
+    if (!excludedFiles.find(item => item === behaviorFileName)) {
+      test(`${behaviorFileName} : File is missing specification tag.`, () => {
+        fail(
+          `File should have specification tag. If tests are written in separate file then add behavior file name into 'excludedFiles'.`,
+        )
+      })
+    }
+  }
+
+  private createMissingDescriptionTest(behaviorFileName: string) {
+    test(`${behaviorFileName} : File is missing description.`, () => {
```

Comment:
maybe would be better to use `Accessibility behavior is missing description` (as this, in fact, what we do care about and is more precise description for failure reason). Lets compare:

- `buttonBehavior.tsx: File is missing description.` - not sure, which description we are talking about? for _file_?? 
- `buttonBehavior.tsx: Accessibility behavior is missing description.` - now it is clear

Reply:
same applies to `fail(`File should have description.`)`